src/js/components/githubUsers/button.tsx

```diff
@@ -0,0 +1,45 @@
+import Button from '@salesforce/design-system-react/components/button';
+import classNames from 'classnames';
+import React from 'react';
+
+import GitHubUserAvatar from '~js/components/githubUsers/avatar';
+import { GitHubUser } from '~js/store/user/reducer';
+
+const GitHubUserButton = ({
+  user,
+  isAssigned,
+  isSelected,
+  ...props
+}: {
+  user: GitHubUser;
+  isAssigned?: boolean;
+  isSelected?: boolean;
+  [key: string]: any;
```

Comment:
This is just saying that the props passed to this button can be any `key: string` pair?

Reply:
That's right -- we can define a few properties we know/care about, then say that it can also have any other arbitrary `string: any` pairs.